### DISCUSSION TITLE
refactor(audio-player): move lightdom styles to shadow dom

### DISCUSSION THREAD

**Discussion by bennypowers** [RedHat-UX/red-hat-design-system#1230] [General]
There are a number of styles in the audio player lightdom sheet which could be moved to shadow dom stylesheets, keeping declarative shadow dom in mind. We should move as many of those styles over as we can.

**Comment by markcaron** [comment]
Converting this to a discussion, because I think we need to formulate a team consensus / stance on light DOM among other questions like CLS, authoring experience, progressive enhancements and accessibility. We also need to consider what we can support today vs paving the way forward with things like DSD.

CC: @nikkimk
